content/components/banner.mdx

```diff
@@ -8,15 +8,254 @@ tags:
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="warning banner"
+    src="https://github.com/primer/design/assets/813754/fc82519b-60e1-45b3-a0de-c5f89108e96e"
+  />
+  <Box as="p" mt="0">
+    <h4>Warning</h4>
+    Use warning banners to alert users about potential issues or impactful changes. For example possible connectivity concerns or hidden items due to active filters.
+  </Box>
+</Box>
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="success banner"
+    src="https://github.com/primer/design/assets/813754/fe04a355-d343-4a79-9948-efc64d6de76e"
+  />
+  <Box as="p" mt="0">
+    <h4>Success</h4>
+    Use success banners to indicate task completion or successful actions. For example to confirm saving of settings or successful issue transfer. Only use success messages if the success is not easily apparent in the UI.
+  </Box>
+</Box>
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="info banner"
+    src="https://github.com/primer/design/assets/813754/172e3eb4-35ea-4d75-a015-6d00d3ebd914"
+  />
+  <Box as="p" mt="0">
+    <h4>Info</h4>
+    Use info banners to inform about non-critical context and information. For example feature prompts or ongoing processes.
+  </Box>
+</Box>
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="upsell banner"
+    src="https://github.com/primer/design/assets/813754/48b37115-c0e0-40c3-83f1-3e831178f175"
+  />
+  <Box as="p" mt="0">
+    <h4>Upsell</h4>
+    Use upsell banners to inform the users about a feature that can be enabled by upgrading their plan.
+  </Box>
+</Box>
+
+### Title
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="banner without title and banner with title"
+    src="https://github.com/primer/design/assets/813754/c5876803-7b5b-486d-b2ec-27fcfdf0c5d8"
+  />
+  <Box as="p" mt="0">
+    <h4>Title</h4>
+    Use a title to consisly present the topic of the banner when the message of the banner is complex and too long to scan.
+  </Box>
+</Box>
+
+### Dismissable
+
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="banners with a dismiss button"
+    src="https://github.com/primer/design/assets/813754/e9863c48-cf3a-472b-a066-0ba139a633ca"
+  />
+  <Box as="p" mt="0">
+    <h4>Dismiss button</h4>
+    Banners with an <i>onDismiss</i> property automatically show a dismiss button.
```

Comment:
```suggestion
    Banners with an `onDismiss` property automatically show a dismiss button.
```

Reply:
Maybe we should explain why you would or wouldn't show the dismiss button?

Reply:
I used italics because for me the backticks don't result in code blocks within the `<Box>` component. Am I missing something?

> Maybe we should explain why you would or wouldn't show the dismiss button?
Yes, great point. Do you already have something in mind here?

Reply:
If you look through the decision making you logged around close buttons (like maybe in the epic updates?) I think you might find something from accessibility conversations? I don't have the answer here off the top of my head. If its an option to not show it, we should have a reason why we offer that and then explain is 😄

Reply:
Hey, I didn't find the section you mention. It is also not really an a11y concern I think.

I added some more context to the section though.